src/oclif/hooks/init.ts

```diff
@@ -148,31 +149,26 @@ export const onError = async (e: any) => {
         }
         log.debug(e)
     }
-  } else {
-    switch (e.name) {
-      case CommandError.name:
-        if (e.message && e.message !== '') {
-          log.error(e.message)
-        }
-        break
-      case SSEConnectionError.name:
-        log.error(e.message ?? 'Connection to login server has failed')
-        break
-      default:
-        log.error('Unhandled exception')
-        log.error('Please report the issue in https://github.com/vtex/toolbelt/issues')
-        log.error('Raw error: ', e)
+  } else if (ErrorReport.isFlowIssue(e)) {
+    if (e.message && e.message !== '') {
```

Comment:
Is this checking `e.message !== ''` necessary?

Reply:
This was necessary in the old toolbelt login flow, but with the new one I don't know exactly if it's necessary or not. Napoli can say a little bit more about it